### PR TITLE
added sar usecase for dask chunks

### PR DESCRIPTION
**Pull request by cpranav93**
Added an additional use-case with mobyle sar example with the idea of exploring dask chunks and how it affects performance. We need to keep in mind that this is not a simple use-case and would require some effort from the students. We need to make this clear when they try this out. I would suggest that this be the use-case that people do when they are done with phenology. @meiertgrootes or @fnattino or @rogerkuou, could someone take a quick look and merge it if its fine?

### CODE REVIEW

**Comment by fnattino** [comment]
Hi @cpranav93, thanks! Looks nice, but I am a bit concerned that it might be quite challenging for anyone that is not very experienced with SAR datasets (a lot of concepts, e.g. SLC, mrm, azimuth and range, may not mean anything to non experts?). We could include it and let people decide (however, I am not sure how helpful I could be myself on this if there will be questions 😅 )

**Comment by meiertgrootes** [comment]
Looks good so far @cpranav93 I was checking it out. @fnattino I think it should be ok. The very basic concepts are simple enough, At the end of the day itt's radar reflectance in a coordiante system which requires transformation. I like the preamble which highlights the goal (dask chunks) a lot. The rest is just flavour to drive home the point that it has real world applicaions...

File: practice-use-cases/usecase2-sar_dask_chunks.ipynb [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Challenge:\n",
+    "\n",
+    "Rerun the above workflow for different chunk sizes and analyse the performance of the dask cluster. See if you can identify an optimal chunk size for this problem for the fixed worker size.\n",
+    "\n",
+    "Hint: Use %%timeit at the top of a jupyter cell to time the execution time for the cell"
```
Comment (fnattino) [inline practice-use-cases/usecase2-sar_dask_chunks.ipynb:170]: `%%timeit ` runs a cell multiple times and return the average execution time (with std). When running things with dask things might go wrong, as intermediate results might be cached, thus leading to much shorter timings when calculations are rerun. So maybe using `%%time` here  works better?

Comment (cpranav93) [reply practice-use-cases/usecase2-sar_dask_chunks.ipynb]: Fixed

**Comment by cpranav93** [comment]
I would agree with Meiert that the idea is to focus on dask. I have given as many hints as possible in the notebook to help them figure out the flow. Otherwise we could also help them out as needed. Merging this now.

**Pull request merged**